Homework3_ZhukAlexander/task1_fizzbuzz.py

```diff
@@ -0,0 +1,15 @@
+# Печатает цифры от 1 до 100
+# вместо кратных 3 пишет FIzz
+# вместо кратных 5 пишет Buzz
+# вместо одновременно кратных 3 и 5 пишет
+# FizzBuzz
+
+for num in range(1, 101):
+    if not num % 3 and not num % 5:
```

Comment:
15?

Reply:
15 multiple at the same time 3 and 5
15 % 3 == 0
15 % 5 == 0

Reply:
if not num % 15 in first conditions much more explicit and do the same